joint_calling/upload_processor.py

```diff
@@ -0,0 +1,110 @@
+""" This function prepares gVCF's uploaded to GCS, based off sample status
+    logged in a database, for further QC and downstream analysis.
+    The upload processor will determine when samples should be added
+    to existing MatrixTables where appropriate and which MatrixTables
+    they should be combined with in this case. Following a successful
```

Comment:
Where did we actually end up on this point -- do we want to keep gVCFs in standard storage indefinitely, together with the MatrixTables?

Reply:
Thanks for raising, Leo. Not sure at present, will get back to you! :)